### DISCUSSION TITLE
Scrolling parent when at the bottom of select scroll

### DISCUSSION THREAD

**Discussion by chrisbrooks** [JedWatson/react-select#5259] [Q&A] [answered]
Is there a way to scroll the parent once you have scrolled to the bottom of the select options list. Currently you scroll to the bottom of the select list but then the parent doesn't continue to scroll. You can replicate this behaviour on the main react select site. https://react-select.com/home

Essentially want this behaviour in the video, where the parent scrolls when you reach the bottom of the dropdown list.
[screen-capture (2).webm](https://user-images.githubusercontent.com/5603144/177665162-6148521e-6839-48a0-9deb-3424253b22d0.webm)

**Comment by Methuselah96** [comment, marked as answer]
Have you tried using `captureMenuScroll={false}`?

**Reply by chrisbrooks** [reply]
thanks much appreciated
